### ISSUE TITLE
Top level packages should have nicer first lines in the docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7869]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7869|
|Type|enhancement|
|Created|2015-04-25 14:51:49Z|
|Branch|https://github.com/twisted/twisted/tree/nicer-tl-docstrings-7869|

When looking at the API documentation, this is generally your first impression of Twisted's docs -- and right now it isn't great.

These should be improved.

<details><summary>Searchable metadata</summary>

```
trac-id__7869 7869
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__ 
branch__branches_nicer_tl_docstrings_7869 branches/nicer-tl-docstrings-7869
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1429973509409840 1429973509409840
changetime__1430101708165524 1430101708165524
version__None None
owner__hawkowl hawkowl
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [44558]) Branching to nicer-tl-docstrings-7869.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl removed owner|
|-|-|

This branch improves and makes the first line of the current top level projects/modules more consistent. It also removes some top level "maintainer" information in the docstrings (which either hasn't been true in a while or isn't really useful) and "future plans" (which is what trac is for).

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @hawkowl|
|-|-|

Thanks. Looks good except for the change in `twisted/web/__init__.py`, why get rid of these api links?

Because some historic data (like author info) is removed I would put it back up for review to make sure at least someone else agrees to make this change.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl removed owner|
|-|-|

Hmm, yes, that is better to have. I've put it back.

Putting up for review again as requested.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Changes looks good.

Without lacking respect I think that is OK to remove the maintainer info.
I don't think that maintainer info is up to date.

+1 for removing todo items from source code... as noted they should be in Trac.

I don't want to ask you to check if each removed work item has an associated ticket. I guess that if the work item is important enough it has or will have a ticket.

If we want to keep a maintainer info in the source code I think that we should start by documenting how this info is supposed to be use.
What does a maintainer mean? Why the other subprojects have a maintainer?
I could not find info on the wiki or twisted docs. 

Good to merge.

Many thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44575]) Merge nicer-tl-docstrings-7869: Nicer top-level docstrings.

Author: hawkowl
Reviewer: thijs, adiroiban
Fixes: [#7869](https://github.com/twisted/twisted/issues/7869)
